src/k8s-extension/azext_k8s_extension/_format.py

```diff
@@ -34,23 +34,28 @@ def __get_table_row(result):
 
 
 def k8s_extension_types_list_table_format(results):
-    return [__get_extension_type_table_row(result, False) for result in results]
+    return [__get_extension_type_table_row(result) for result in results]
 
 
 def k8s_extension_type_show_table_format(result):
-    return __get_extension_type_table_row(result, True)
+    return __get_extension_type_table_row(result)
 
 
-def __get_extension_type_table_row(result, showReleaseTrains):
+def __get_extension_type_table_row(result):
     # Populate the values to be returned if they are not undefined
-    clusterTypes = ', '.join(result['clusterTypes'])
+    clusterTypes =''
+    if result['properties']['supportedClusterTypes'] is not None:
+        clusterTypes = ', '.join(result['properties']['supportedClusterTypes'])
+
     name = result['name']
     defaultScope, allowMultipleInstances, defaultReleaseNamespace = '', '', ''
-    if result['supportedScopes']:
-        defaultScope = result['supportedScopes']['defaultScope']
-        if result['supportedScopes']['clusterScopeSettings']:
-            allowMultipleInstances = result['supportedScopes']['clusterScopeSettings']['allowMultipleInstances']
-            defaultReleaseNamespace = result['supportedScopes']['clusterScopeSettings']['defaultReleaseNamespace']
+    if result['properties']['supportedScopes']:
+        defaultScope = result['properties']['supportedScopes']['defaultScope']
+        if result['properties']['supportedScopes']['clusterScopeSettings'] is not None:

```

Comment:
>if result['properties']['supportedScopes']['clusterScopeSettings'] is not None: [](http://example.com/codeflow?start=8&length=79)

Any reason why you are not handling 'namespacescopesettings'?

Reply:
I just tweaked an existing function to work with the current data format. Not sure if we want namespace scoped settings to be viewed in the table format

Reply:
It'd be good to be consistent.  Can you pl. add that too?